horizons/util/loaders/sqliteanimationloader.py

```diff
@@ -30,22 +30,13 @@ class SQLiteAnimationLoader:
 	def __init__(self):
 		pass
 
-	def loadResource(self, location):
+	def loadResource(self, anim_id):
 		"""
-		@param location: String with the location. See below for details:
-		Location format: <animation_id>:<command>:<params> (e.g.: "123:shift:left-16, bottom-8)
-		Available commands:
-		- shift:
-		Shift the image using the params left, right, center, middle for x shifting and
-		y-shifting with the params: top, bottom, center, middle.
-		A param looks like this: "param_x(+/-)value, param_y(+/-)value" (e.g.: left-16, bottom+8)
-		- cut:
-		#TODO: complete documentation
+		@param anim_id: String with the location
 		"""
-		commands = location.split(':')
-		anim_id = commands.pop(0)
+
+		anim_id = location
```

Comment:
I might be missing something, but where does `location` come from? It was removed in favor of `anim_id`, correct?

Reply:
Oops. I should have removed that line, yes.
Turns out I haven't actually tested this class yet because it is only used as fallback.